wp-content/themes/pub/wporg-learn-2020/css/vendor/_sensei.scss

```diff
@@ -144,3 +144,6 @@ details {
 		line-height: 2.8;
 	}
 }
+.wp-block-sensei-lms-course-results.is-style-default .wp-block-sensei-lms-course-results__module-header:not(.has-background) {
+    background-color: #eee
```

Comment:
Isn't this line ended with a semi-colon (;) in the last? For the consistent code practice according to the file.

Reply:
thanks - I've updated this